drivers/classic/src/mtclim_vic.c

```diff
@@ -38,16 +38,11 @@ int
 data_alloc(const control_struct *ctrl,
            data_struct          *data)
 {
-    int ok = 1;
-    int ndays;
+    int    ok = 1;
```

Comment:
consider changing to 

```C
bool ok = true;
```

Since that is closer to what we mean here. However, it's OK as is, so leaving as is is fine too.